cleaner/cleaner.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2022.
```

Comment:
Shouldn't the license contians Red Hat, Inc?

something like `Copyright 2019 Red Hat, Inc. and/or its affiliates`

Reply:
ok